.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- The title should folllow Angular Commit Message Conventions (https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) -->
+
+**What is the previous behavior?**
+
+
+**What is the new behavior?**
```

Comment:
I wonder if we should have a general Summary / What this PR does before these two more specific questions?  Not sure, maybe the title is enough...